src/bilance_queue.py

```diff
@@ -58,19 +58,23 @@ def put(self, item: Union[Operation, BilancedOperation]) -> None:
         for fee in buffer_fee:
             self.remove_fee(fee)
 
-    def get(self) -> BilancedOperation:
+    def get(self) -> Union[BilancedOperation, None]:
```

Comment:
We should use Optional[object] instead of Union[object, None].

Reply:
Alright. I have to admit I don't use type annotations that often. Learning never stops! :)